Sources/DistributedActors/DistributedActors.docc/Clustering.md

```diff
@@ -273,19 +273,55 @@ distributed actor Boss: LifecycleWatch {
 
 Remote calls are at the heart of what makes distributed actors actually distributed.
 
-A call made on a remote distributed actor reference, will cross network boundaries, and therefore may way due to 
+A call made on a remote distributed actor reference will cross network boundaries, and therefore may fail due to 
 network issues, message loss, serialization errors, or other reasons such as the recipient node crashing as it 
 processes the message. Even replies to remote calls could sometimes fail being delivered, so you might need to 
 design your distributed actors with idempotency (the resilience of a method being called more than once, e.g. due to a retry) in mind.
 
 By default, to avoid "hanging" a remote caller forever on a suspended remote call as the recipient node fails to reply to it,
-for example because it (or the network itself), are currently unresponsive, remote calls have a default timeout configured,
-and if no reply is received within this duration, the call will fail with a ``RemoteCallError``.
+for example because it (or the network itself), is currently unresponsive, remote calls have a default timeout configured.
+If no reply is received within this duration, the call will fail with a ``RemoteCallError.timedOut``.
```

Comment:
weirdly enough docc syntax is ``RemoteCallError/timedOut`` heh, will fix later :)

Reply:
ah. fixed in https://github.com/apple/swift-distributed-actors/pull/1020/commits/c48d49f81c1583f89e40dad78b3c375ca7455b81